dockerfiles/run/guice/cassandra-ldap/destination/conf/listeners.xml

```diff
@@ -19,8 +19,6 @@
  -->
 
 <listeners>
-  <poolSize>8</poolSize>
```

Comment:
no more needed, until we are in trouble again and want to tweak things (:

Reply:
What's you plan regarding this?

Shouldn't it be more "Tweaking Reactor elastic scheduler" ?

Reply:
I don't know for now, we can reintroduce it later, I don't have problem with that, I'm just wondering if we are not overly optimistic